apps/frontend-e2e/src/e2e/app.cy.ts

```diff
@@ -1,13 +1,35 @@
-import { getGreeting } from '../support/app.po'
-
+/* eslint-disable cypress/no-unnecessary-waiting */
```

Comment:
I know its better to create a map of some sort from filename to stats and use that. But, its a start, will raise another PR to fix the e2e